### DISCUSSION TITLE
Automate  the remove of data in the PR_OST_OSTID binary property

### DISCUSSION THREAD

**Discussion by YuvalGrMic** [microsoft/mfcmapi#497] [Q&A] [answered]
Hey,
lately, I came across a problem in my environment that lead me to the use of MFCAMP:  
https://docs.microsoft.com/en-US/outlook/troubleshoot/performance/cannot-start-outlook-in-cached-mode-or-create-profile
The solution is basically to remove the binary data in the PR_OST_OSTID property.
I am now trying to write a batch file to do it automatically and thinking of using MrMAPI for that but I am not managing to clear the Binary Data from the property.
Is it possible? if Yes It can really help if you have a code example that you can share that is doing similar actions.
thanks in advance!

**Comment by stephenegriffin** [comment, marked as answer]
mrmapi has no write capabilities exposed
